lockfile/src/spdx.rs

```diff
@@ -272,42 +291,6 @@ mod tests {
         }
     }
 
-    #[test]
```

Comment:
Test is removed as this is now an `UnknownEcosystem`.

Reply:
https://github.com/phylum-dev/cli/blob/2aee6488a96046716886c39d02fe7d03e82c3a09/lockfile/src/spdx.rs#L161

We just let those pass. I could make a test that has has a list of know and unknown and show they aren't captured?

Reply:
Yeah if you have another test case that is supposed to test for the succesful case we could probably add a package that covers this behavior to the fixture. I just think we want that behavior covered somewhere.

Reply:
I'm confused; we were talking about missing test coverage, not a behavior change?

Reply:
The test was originally removed due to behavior change. The behavior being that this used to fail parsing if we didn't have an external reference with package locator or a PURL to determine the triplet (package name, version, ecosystem). If we came across an unknown ecosystem with the external reference, we gave a warning and continued to parse the remaining packages.

Now that this uses the download location to determine the ecosystem, this test should no long fail regarding missing a PURL or locator since we can fallback on using the downloadlocation. If it's a url from a known registry then we can determine enough information to send off for processing, otherwise it is now an unknown ecosystem.

Reply:
Yes, I was just saying that we should have _some_ test or even just a singular package in another one of the tests that shows that if you get an `UnknownEcosystem`, then the whole process doesn't fail.

Reply:
Ah gotcha, it's covered by the current [test](https://github.com/phylum-dev/cli/blob/d8e80a11ec38cc690f6251b75b801c6b27c8b53b/lockfile/src/spdx.rs#L370). The parsing comes back empty without failing if only one package and it's an `UnknownEcosystem`